requirement/src/main/java/org/arquillian/cube/requirement/Requirements.java

```diff
@@ -26,4 +33,24 @@ public static void checkRequirement(Requires requires, Annotation context) throw
             }
         }
     }
+
+    private static Requirement getRequirement(Annotation context) {
```

Comment:
Maybe some better method name suggesting it's dynamically looking for SPI implementation?